### PR TITLE
Update for 0.15

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

File: app/src/main/java/io/appwrite/playgroundforandroid/MainActivity.kt [hunk 1]
```diff
@@ -90,9 +90,6 @@ class MainActivity : AppCompatActivity() {
         binding.updatePrefs.setOnClickListener {
             viewModel.updateAccountPrefs()
         }
-        binding.deleteAccount.setOnClickListener {
-            viewModel.deleteAccount()
-        }
```
Comment (lohanidamodar) [inline app/src/main/java/io/appwrite/playgroundforandroid/MainActivity.kt:95]: You could use `updateStatus`?

File: app/src/main/java/io/appwrite/playgroundforandroid/PlaygroundViewModel.kt [hunk 2]
```diff
@@ -20,9 +21,12 @@ import java.lang.System.currentTimeMillis
 
 class PlaygroundViewModel : ViewModel() {
 
-    private val collectionId: String = "YOUR_COLLECTION_ID" // Single required 'username' string attribute
-    private val functionId: String = "YOUR_FUNCTION_ID"
-    private val bucketId: String = "YOUR_BUCKET_ID"
+    private val endpoint = "http://192.168.4.139/v1"
```
Comment (lohanidamodar) [inline app/src/main/java/io/appwrite/playgroundforandroid/PlaygroundViewModel.kt:24]:
```suggestion
    private val endpoint = "YOUR_ENDPOINT"
```

**Review by lohanidamodar — APPROVED** [review]
Looks good, quick comment.

File: app/src/main/java/io/appwrite/playgroundforandroid/MainActivity.kt (hunk 1 above)
Comment (abnegate) [reply app/src/main/java/io/appwrite/playgroundforandroid/MainActivity.kt]: Good plan, have updated now

**Pull request merged**